xapian-applications/omega/templates/clicklog

```diff
@@ -0,0 +1,3 @@
+$httpheader{Status,302 Found}
+$httpheader{Location,$cgi{URL}}
+$log{clicks.log,$cgi{QUERYID}	$cgi{DOCID}}
```

Comment:
Is there a reason not to include a newline at the end? Some editors will add one on save, which will introduce unwanted changes in future.

I also wonder if it'll be clear to anyone editing that you're using a tab character. (It will depend on whether their editor highlights it, I suspect.) CSV would work fine here since we're dealing with numbers.